components/content_settings/core/browser/brave_content_settings_pref_provider.h

```diff
@@ -47,6 +49,13 @@ class BravePrefProvider : public PrefProvider,
       bool incognito) const override;
 
  private:
+  friend class BravePrefProviderTest;
+  FRIEND_TEST(BravePrefProviderTest, TestShieldsSettingsMigration);
```

Comment:
FRIEND_TEST_ALL_PREFIXES (useful e.g. for disabling tests: `DISABLED_TestShieldsSettingsMigration)

Reply:
Fixed in bdc2851b5386b1ae42a134550a217edb6267ef57